modules/scrabble.js

```diff
@@ -120,7 +120,10 @@ module.exports = {
       phrase.forEach((word) => {
         word = word.replace(/[^A-Za-z]+$/, '')
         let result = computeWord(word)
-        if (wordHistory.includes(result.word) || !result.isPossible) return
+        console.log(wordHistory.map(word => word.toUpperCase()))
```

Comment:
Can we get rid of these console.logs?
We can add them back in once we do #279 in 2025 or something

Reply:
Oh yeah, that was me testing things. I'll fix that.